Dockerfile

```diff
@@ -70,7 +70,9 @@ ENV JAVA_HOME=/usr/lib/jvm/java-${JAVA_MAJOR}-openjdk
 ENV LANG C.UTF-8
 ENV LC_ALL C.UTF-8
 
-RUN apk add openjdk${JAVA_MAJOR}-${JDIST}-headless
+RUN apk update && \
```

Comment:
we can use the short form `apk -U upgrade` and add `--no-cache` to all commands or add `rm -vrf /var/cache/apk/*` after to save a few MB of package index - see: https://newbedev.com/alpine-dockerfile-advantages-of-no-cache-vs-rm-var-cache-apk